### ISSUE TITLE
Create task with only date?

### ISSUE THREAD

**Issue by mariomaz87** [HeroBlackInk/ultimate-todoist-sync-for-obsidian#94]
Hi,
Maybe I’m missing something, but can I create a task with date but no time? When I do, in Todoist the time is set to 11:59 am. 

Thanks!

**Comment by eudennis** [comment]
Just to be sure, are you not using the fork I did? Because that was a known bug that I believe is fixed on the latest version already.

https://github.com/eudennis/ultimate-todoist-sync-for-obsidian-experiment

**Comment by mariomaz87** [comment]
> Just to be sure, are you not using the fork I did? Because that was a known bug that I believe is fixed on the latest version already.
> 
> https://github.com/eudennis/ultimate-todoist-sync-for-obsidian-experiment

Yes, I’m now using your fork and the bug has been fixed in the fork, thanks!
